go.mod

```diff
@@ -20,13 +20,13 @@ require (
 	github.com/stretchr/testify v1.8.0
 	gomodules.xyz/jsonpatch/v2 v2.2.0
 	helm.sh/helm/v3 v3.10.3
-	k8s.io/api v0.25.3
-	k8s.io/apiextensions-apiserver v0.25.3
-	k8s.io/apimachinery v0.25.3
-	k8s.io/cli-runtime v0.25.3
-	k8s.io/client-go v0.25.3
-	k8s.io/utils v0.0.0-20221012122500-cfd413dd9e85
-	sigs.k8s.io/controller-runtime v0.13.0
+	k8s.io/api v0.26.1
+	k8s.io/apiextensions-apiserver v0.26.1
+	k8s.io/apimachinery v0.26.1
+	k8s.io/cli-runtime v0.26.1
+	k8s.io/client-go v0.26.1
+	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
+	sigs.k8s.io/controller-runtime v0.14.4
 	sigs.k8s.io/kubebuilder/v3 v3.7.0
```

Comment:
Are we not bumping Kubebuilder?

Reply:
v3.7.0 Kubebuilder version is already pointing to k8s 1.26. I don't think it is necessary. I am planning to raise PR for the **java-plugin** similarly.

Reply:
need to update it to v3.9.0 It has k8s bump of 1.26. I will update the java plugin similarly.

Reply:
Bumped KB to `v3.9.0`.